finalProject/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Arthur Tomasi Slide Puzzle</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
+</head>
+    <body>
+        <img  id="test" src="assets/3x3.jpg">
+        <h1>Final Project - Programming Languages Exercices</h1>
+        <div id="optionBarTop">
+            <div class="shuffle-container">
```

Comment:
for the folders structure, create subfolders inside `assets`: `images`, `fonts`, `sounds`. Put the related files inside